documentation/API_documentation/QoD_Stable_Latency_API.md

```diff
@@ -0,0 +1,181 @@
+# Overview
+
+<span class="colour" style="color:rgb(0, 0, 0)">The Quality-On-Demand (QoD) API for stable latency provides programmable interface for developers and other users (capabilities consumers) to specify stable data-transfer latency managed by Telco networks without necessity to have in-depth knowledge of the 4G/5G system complexity by abstracting the internal complexity of telecom systems [1].</span>
+
+## 1\. Introduction
+
+There are many real-world use cases which could exploit the potential of programmable networks. Several of these use case implementations are still work in progress or have not been operationalized as developers have no means to configure or request stable latency or bandwidth during congestion from a mobile network operator which is critical. Use cases from industrial (IoT), VR/Gaming, broadcasting, autonomous driving and many others fall in such a category since they demand network communication quality and are sensitive to any change in transmission conditions.
+
+The QoD latency API offers the application developers and users the capability to request for stable latency for a specified App-Flow(s) between application clients and backend services (Application Servers). The developer has a pre-defined set of QoS\_Profiles which he could choose from depending on his latency requirements.
```

Comment:
"The QoD latency API offers the application developers and users the capability to request for stable latency for a specified App-Flow(s) between application clients and backend services (Application Servers)." -> "The QoD latency API offers to application developers the capability to request stable latency for a specified application flow between User Equipment and Application Servers."

Reply:
Changed within [d93c9fc](https://github.com/camaraproject/QualityOnDemand/pull/18/commits/d93c9fca38d033602f860cad70e14ce2633340c6) to"The QoD latency API offers the application developers the capability to request for stable latency (reduced jitter) for a specified App-Flow between User Equipment (application clients) and Application Servers (backend services)." ... there was also a request to explain User Equipment and Application Servers.